nana/modules/database/anime_db.py

```diff
@@ -0,0 +1,62 @@
+import threading
+from nana import SESSION, BASE
+from sqlalchemy import Column, UnicodeText, Numeric
+
+
+class Favourites(BASE):
+    __tablename__ = "favourites"
+    user_id = Column(Numeric, primary_key=True)
+    data = Column(UnicodeText, primary_key=True)
+
+    def __init__(self, user_id, data):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)